### ISSUE TITLE
Markdown tables broken?

### ISSUE THREAD

**Issue by Tekl** [typemill/typemill#227]
I've a table like this:

```
| Name | amer./brit. | Lautschrift | Aussprache | Wortart | Sprachgebrauch | Fachgebiet | Beispiele | 
|---|---|---|---|---|---|---|---|
| BeoLingus | **ja** | nein | nein | **ja** | **ja** | **ja** | nein | 
| Oxford | **ja** | **ja** | nein | nein | nein | nein | **ja** | 
| Wiktionary | nein | **ja** | **ja**[^1] | **ja** | nein | nein | nein | 
| dict.cc (Lipflip) | **ja**[^2] | nein | nein | **ja** | *teilweise* | nein | nein | 
| dict.cc (Tekl) | **ja**[^2] | nein | nein | **ja** | nein | ja | new | 
```

In the Editor I just see plain text:

<img width="822" alt="Safari_190346" src="https://user-images.githubusercontent.com/3667947/97613869-83a40080-1a19-11eb-980a-f6d268bb3068.png">

When I try to edit it, the Editor recognizes the table but replaces the headers with dashes. Saving this edit will destroy my content.

<img width="850" alt="Safari_190357" src="https://user-images.githubusercontent.com/3667947/97613904-8e5e9580-1a19-11eb-9edc-7829afe50225.png">

**Comment by trendschau** [comment]
thank you, I somehow did not notice that, but I see some more problems e.g. with definition lists. The reason is that I switched to the parsedown-library in order to detect blocks more accurately. It is not perfect yet, I will try to fix for next release.

**Comment by trendschau** [comment]
fix published with version 1.4.1
